checkov/main.py

```diff
@@ -628,6 +629,15 @@ def run(self, banner: str = checkov_banner) -> int | None:
             # if a bc_api_key is passed it'll save it.  Otherwise it will check ~/.bridgecrew/credentials
             self.config.bc_api_key = bc_integration.persist_bc_api_key(self.config)
 
+            if not self.config.bc_api_key:
```

Comment:
is it possible that it will fail jenkins because of the new warning ?

Reply:
sure, this is intentionally, because you basically get no results, if a user adds a severity to the check filtering without adding an API key 🙂